heal-remedy.lic

```diff
@@ -0,0 +1,93 @@
+# quiet
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#tendme
+=end
+
+custom_require.call(%w[common common-healing common-items])
+
+
+arg_definitions = [
+  [
+    { name: 'debug', regex: /debug/i, optional: true }
+  ]
+]
+
+args = parse_args(arg_definitions)
+$debug_mode_hr = args.debug
+
+$remedies = get_data('remedies')
+
+def wounds
+  wounds = DRCH.check_health
+  pause 1
+#  echo (wounds)
```

Comment:
Do you want to uncomment this but make it conditional on `$debug_mode_hr`?

Reply:
I did.